src/Widgets/TerminalWidget.vala

```diff
@@ -46,6 +46,7 @@ namespace PantheonTerminal {
                 this.app = value.app;
                 this.menu = value.menu;
                 this.menu.show_all ();
+                this.menu.get_children ().nth_data (0).set_visible (false);
```

Comment:
This relies on a certain menu order.  I would just disable the linked action and not hide the option.

Reply:
I hide it because I thought the I would be annoying to have an almost always disabled menu item, so it's just the first one if you have a link, or it won't appear

Reply:
I don't know they only idea I came up with is storing this menu item but it's not a good idea too :(

Reply:
In Files this problem is solved by building the context menu on the fly each time it is required since the context varies a lot. I'd have to study the code in more detail to see whether that is appropriate here though.

Reply:
I think editing the "Show in File Browser" is a better idea, I will just wait for the UX team to comment on this as I won't be able to edit anything today :D

Reply:
No, the "Show in File Browser" item relates to the whole view and not what's being clicked on specifically; that's why it's separated below the text items. An "Open in Browser" item should probably be grouped along with the text-related items up above. I'm not super picky about whether it's insensitive or hidden when it's irrelevant; I feel like for context menus when you're clicking on a specific item it makes sense to only show items relevant to that item, i.e. not showing the "Open in Browser" item at all unless a link is being clicked. But if it greatly simplifies things to always show it and only toggle sensitivity, I think we'd want it to be at the bottom of the text-relted itms, at least.

@danrabbit thoughts?

Reply:
There is precedent for the same menuitem applying to either an item or the background depending on the context - e.g. the "Open in" menuitem in Files.